pkg/security/api/types.go

```diff
@@ -5,6 +5,11 @@ import (
 	"k8s.io/kubernetes/pkg/api/unversioned"
 )
 
+// +genclient=true
+
+// Security is just a name for the client generated
+type Security struct{}
```

Comment:
This seems to generate a client that will always fail.  What is the intent for this?